src/views/Messenger.vue

```diff
@@ -134,6 +134,10 @@ export default {
   },
 
   computed: {
+    isChannelPanelPined () {
+      return this.uiPinChannelSidePanel || this.$route.path === process.env.BASE_URL
```

Comment:
Is this really the best way to check the current route @HacksOnNet?

Reply:
You're right; I changed it to name checking.